### PR TITLE
[PT/XLA] Add many more ResNet50 tests

### PR DESCRIPTION
**Pull request by will-cromar**
# Description

Porting several existing variants of this test to the new system.

@JackCaoG let me know if there's another nightly variant that you want to add here.

# Tests

Checked DAG construction:

<details>

image

</details>

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: dags/pytorch_xla/pytorchxla_torchvision.py [hunk 1]
```diff
@@ -78,7 +100,8 @@
       US_EAST1_C,
   ).run()
 
-  mnist_v2_8 >> (resnet_v2_8, resnet_v4_8, resnet_v5lp_4)
+  mnist_v2_8 >> (resnet_v2_8, *resnet_v4_8_tests, resnet_v4_32, resnet_v5lp_4)
+  resnet_v2_8 >> resnet_v3_8_tests
 
   resnet_v100_2x2 = task.GpuGkeTask(
```
Comment (RissyRan) [inline dags/pytorch_xla/pytorchxla_torchvision.py:106]: We probably want to add post_process sooner to multihost GPU tests, as our dashboard summary is missing the track of those [tests](http://go/ml-auto-solutions-dashboard).

**Review by RissyRan — APPROVED** [review]
Thanks Will! The screenshot is nice, but I will probably prefer to have a link of the test that could direct me to original tests (composer env).

File: dags/pytorch_xla/pytorchxla_torchvision.py (hunk 1 above)
Comment (will-cromar) [reply dags/pytorch_xla/pytorchxla_torchvision.py:106]: I can add this once I'm done porting tests for the 2.3 release.

**Comment by will-cromar** [comment]
> Thanks Will! The screenshot is nice, but I will probably prefer to have a link of the test that could direct me to original tests (composer env).

Makes sense. I skipped actually running any of these in this case because there are too many and these tests have already been running in the old system for a while. We also know that other variants have been passing in airflow.

**Pull request merged**